Replaced the LC with the TSC.

dblock opened the pull request:
### Description

The OpenSearch Project is supported by [The OpenSearch Software Foundation](https://foundation.opensearch.org/), a project of [The Linux Foundation](https://www.linuxfoundation.org/). You can read the launch announcement [here](https://www.linuxfoundation.org/press/linux-foundation-announces-opensearch-software-foundation-to-foster-open-collaboration-in-search-and-analytics) and learn more about joining the foundation [here](https://foundation.opensearch.org/). 

The project has a [Governing Board](governing-board/README.md) and a [Technical Steering Committee](technical-steering-committee/README.md).

This PR adds the initial TSC charter and creates a placeholder for the GB. 

Closes https://github.com/opensearch-project/community/issues/48. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

krisfreedain commented:
thank you @dblock - one comment - completely erasing the records of the LC feels inappropriate for transparency - I'd say we should likely note that it is super-ceded by TSC at least for a set time-period

dblock commented:
> thank you @dblock - one comment - completely erasing the records of the LC feels inappropriate for transparency - I'd say we should likely note that it is super-ceded by TSC at least for a set time-period

I replaced MEMBERS.md in the LC folder by a note with a link to the last working version with the list of people and added a README with the same to access meeting notes. Does that work?

andrross reviewed (commented):
Looks like a `.DS_Store` file slipped in here

KarstenSchnitter reviewed (approved):
Good work. Only minor remarks.

dblock commented on technical-steering-committee/CHARTER.md:
```diff
@@ -0,0 +1,127 @@
+
+   2. Quorum for TSC meetings requires at least fifty percent of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met but will be prevented from making any decisions at the meeting.
+
+   3. Except as provided in Section 7.c. and 8.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the TSC.
```
Good catch, I lost that in some export / import. I believe this is correct now, check again?

AmiStrn reviewed (approved):
Nothing to add. This is great, Thanks @dblock !

reta commented on technical-steering-committee/CHARTER.md:
```diff
@@ -0,0 +1,126 @@
+
+   b. Startup Period.
+
+      1. The initial voting members ("Initial Members") of the TSC at the inception of the Project (the "Start Date") are those individuals identified as voting members in the "STEERING.md" file within the Project’s .github repository. Initial Members will consist of Initial Members serving one-year terms and Initial Members serving two-year terms, as indicated in the STEERING.md file.
```
I believe we set on TSC to be one year term?

dblock replied:
We always need some continuity, so it's 2 years to create overlap. At start LC members are 1 year so we can re-elect half after year 1, and the rest 2 years so we can re-elect the other half and then it's business as usual.

reta reviewed (approved):
Thanks @dblock !

samuel-oci commented:
Thank you @dblock !

dblock commented:
@krisfreedain Can this be merged?

rbowen commented on technical-steering-committee/CHARTER.md:
```diff
@@ -0,0 +1,126 @@
+
+      8. discussions, seeking consensus, and where necessary, voting on technical matters relating to the code base that affect multiple projects; and
+
+      9. coordinating any marketing, events, or communications regarding the Project.
```
This seems wrong. That's *definitely* not the job of the TSC. Point 9 is the job description of the community manager, and would be a significant distraction from the TSC's correct focus - on the steering the technical direction of the project.

dblock replied:
Thanks @rbowen.

I think this item came from the template we used and is present in other charters under LF, [here](https://openmessaging.cloud/charter/) and [here](https://caseontology.org/resources/charter.html). 

Removing that paragraph would be an amendment to the Technical Charter that was reviewed by the members before we announced the foundation, which requires a 2/3 vote of the entire TSC and approval by LF. I opened https://github.com/opensearch-project/community/issues/53, please add your comments.

dblock commented:
We had a discussion on whether this content belongs in the community repo and decided to create a new **steering** one. @krisfreedain is taking care of it.

I'll split this PR in two then.

dblock commented:
I've moved the TSC parts to https://github.com/opensearch-project/technical-steering (members merged as part of https://github.com/opensearch-project/technical-steering/pull/4).

@krisfreedain take a look again?
